mailpoet/lib/Newsletter/Shortcodes/Categories/Site.php

```diff
@@ -29,9 +29,16 @@ public function process(
       case 'title':
         return $this->wp->getBloginfo('name');
 
-      case 'homepage_link':
+      case 'homepage_url':
         return $this->wp->getBloginfo('url');
 
+      case 'homepage_link':
+        return sprintf(
+          '<a target="_blank" href="%s">%s</a>',
+          $this->wp->getBloginfo('url'),
+          $this->wp->getBloginfo('name')
+        );
+
```

Comment:
@lysyjan are we displaying the result of the shortcodes on the browser?

If yes, are they escaped later on? If they are displayed and not escaped, could we use `esc_url` and `esc_html`?

Please ignore the comment if it doesn't apply to this case.

Reply:
Good point! What I tested was without an issue, but I added escaping functions on both strings in the last commit.